invenio/utils/datastructures.py

```diff
@@ -295,6 +295,9 @@ def __len__(self):
     def keys(self):
         return self._dict.keys()
 
+    def values(self):
```

Comment:
can you please separate this change to a new commit so we can backport it to maint-2.0?